packages/miniflare/src/plugins/core/index.ts

```diff
@@ -65,7 +65,12 @@ if (process.env.NODE_EXTRA_CA_CERTS !== undefined) {
   // https://nodejs.org/api/cli.html#node_extra_ca_certsfile
   try {
     const extra = readFileSync(process.env.NODE_EXTRA_CA_CERTS, "utf8");
-    trustedCertificates.push(extra);
+    // Split bundle into individual certificates and add each individually:
+    // https://github.com/cloudflare/miniflare/pull/587/files#r1271579671
+    const pemBegin = "-----BEGIN";
```

Comment:
Does PEM actually specify exactly 5 `-` and no whitespace? I've always assumed the format was looser than that but I guess I don't know.